src/background/injector-controller.ts

```diff
@@ -42,22 +39,18 @@ export class InjectorController {
             inspectStoreInjectingRequested || visualizationStoreState.injectingRequested;
 
         if (isInjectingRequested && !visualizationStoreState.injectingStarted) {
-            this.setTimeout(() => {
-                this.interpreter.interpret({
-                    messageType: Messages.Visualizations.State.InjectionStarted,
-                    tabId: tabId,
-                });
-            }, InjectorController.injectionStartedWaitTime);
+            this.interpreter.interpret({
```

Comment:
Why do we no longer want to set a timeout here?

Reply:
This was done a long while back to avoid the store "same scope" error that could occur. This can no longer occur with the changes we've already made.